src/D2L.Bmx/ConsoleWrapper.cs

```diff
@@ -0,0 +1,21 @@
+namespace D2L.Bmx;
+
+internal interface IConsole {
+	string? ReadLine();
+	void Write( string? value );
+	void WriteLine( string? value );
+}
+
+internal class ConsoleWrapper : IConsole {
```

Comment:
Don't think this kind of abstraction is necessary.
It would be one thing if it was an application that can serialize data out in different ways (like files vs to std out etc...).
Did you have something in mind that would benefit from this?

Reply:
Hmm I got that abstraction from @cfbao's branch that was used when we discussed bmx's code structure. I should have followed up on the significance of the wrapper before implementing it though. @cfbao is the wrapper needed?

Reply:
Should've clarified - I was just using that as an example of a mockable dependency, not to suggest that we actually want to do this.

I can see its utility if we want to unit test the CLI experience, but probably not worth it given how simple bmx's workflow is.

Reply:
Ok, I have removed all references to ConsoleWrapper